portfolio/src/main/webapp/script.js

```diff
@@ -23,7 +38,13 @@ async function getComments() {
   const ul = document.createElement('ul');
   for (const comment of comments) {
     const li = document.createElement('li');
-    li.innerText = comment;
+    li.innerText = comment.content;
+    if (comment.imageURL != null) {
```

Comment:
Can this be undefined too? Would `if (comment.imageURL)`work better?

Reply:
I did some research and used != null instead of !== because it can be undefined as well. However, your suggestion seems to work well and reads clearly, so I prefer it!